Basic examples fails when updated to v0.11.15

`x86_64-unknown-uefi`-target is missing

```
  [lock_api 0.4.10] error[E0463]: can't find crate for `std`
  [lock_api 0.4.10]   |
  [lock_api 0.4.10]   = note: the `x86_64-unknown-uefi` target may not be installed
  [lock_api 0.4.10]   = help: consider downloading the target with `rustup target add x86_64-unknown-uefi`
  [lock_api 0.4.10]   = help: consider building the standard library from source with `cargo build -Zbuild-std`
  [lock_api 0.4.10] 
  [lock_api 0.4.10] error: aborting due to 1 previous error
  [lock_api 0.4.10] 
  [lock_api 0.4.10] For more information about this error, try `rustc --explain E0463`.
     Compiling rustc_version v0.4.0
       Running `CARGO=/home/pepper/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/cargo CARGO_CRATE_NAME=rustc_version CARGO_MANIFEST_DIR=/home/pepper/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rustc_version-0.4.0 CARGO_MANIFEST_PATH=/home/pepper/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rustc_version-0.4.0/Cargo.toml CARGO_PKG_AUTHORS='Dirkjan Ochtman <dirkjan@ochtman.nl>:Marvin Löbel <loebel.marvin@gmail.com>' CARGO_PKG_DESCRIPTION='A library for querying the version of a installed rustc compiler' CARGO_PKG_HOMEPAGE='' CARGO_PKG_LICENSE=MIT/Apache-2.0 CARGO_PKG_LICENSE_FILE='' CARGO_PKG_NAME=rustc_version CARGO_PKG_README=README.md CARGO_PKG_REPOSITORY='https://github.com/Kimundi/rustc-version-rs' CARGO_PKG_RUST_VERSION='' CARGO_PKG_VERSION=0.4.0 CARGO_PKG_VERSION_MAJOR=0 CARGO_PKG_VERSION_MINOR=4 CARGO_PKG_VERSION_PATCH=0 CARGO_PKG_VERSION_PRE='' LD_LIBRARY_PATH='/tmp/cargo-installpqEK6z/release/deps:/home/pepper/dev/ghrepos/bootloader/examples/basic/target/debug/deps:/home/pepper/dev/ghrepos/bootloader/examples/basic/target/debug:/home/pepper/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib:/home/pepper/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib' /home/pepper/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/rustc --crate-name rustc_version --edition=2018 /home/pepper/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rustc_version-0.4.0/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debug-assertions=off --check-cfg 'cfg(docsrs,test)' --check-cfg 'cfg(feature, values())' -C metadata=22f0e4f99d384568 -C extra-filename=-0da39d057b49ccfc --out-dir /tmp/cargo-installpqEK6z/release/deps -C strip=debuginfo -L dependency=/tmp/cargo-installpqEK6z/release/deps --extern semver=/tmp/cargo-installpqEK6z/release/deps/libsemver-abfdaa925a6a7723.rmeta --cap-lints warn`
  [lock_api 0.4.10] error[E0463]: can't find crate for `core`
  [lock_api 0.4.10]   |
  [lock_api 0.4.10]   = note: the `x86_64-unknown-uefi` target may not be installed
  [lock_api 0.4.10]   = help: consider downloading the target with `rustup target add x86_64-unknown-uefi`
  [lock_api 0.4.10]   = help: consider building the standard library from source with `cargo build -Zbuild-std`
  [lock_api 0.4.10] 
  [lock_api 0.4.10] error: aborting due to 1 previous error
  [lock_api 0.4.10] 
  [lock_api 0.4.10] For more information about this error, try `rustc --explain E0463`.
```

--- a/README.md
+++ b/README.md
@@ -43,7 +43,7 @@ To combine your kernel with `bootloader` and create a bootable disk image, follo
     };
     bootloader_api::entry_point!(kernel_main, config = &CONFIG);
     ```
-- Compile your kernel to an ELF executable by running **`cargo build --target x86_64-unknown-none`**. You might need to run `rustup target add x86_64-unknown-none` before to download precompiled versions of the `core` and `alloc` crates.
+- Compile your kernel to an ELF executable by running **`cargo build --target x86_64-unknown-none`**. You might need to run `rustup target add x86_64-unknown-none` for BIOS and `rustup target add x86_64-unknown-uefi` for UEFI before to download precompiled versions of the `std`, `core` and `alloc` crates.
 - Thanks to the `entry_point` macro, the compiled executable contains a special section with metadata and the serialized config, which will enable the `bootloader` crate to load it.
 
 #### Creating a bootable image
@@ -82,7 +82,7 @@ To combine your kernel with `bootloader` and create a bootable disk image, follo
       # rust-toolchain.toml
       [toolchain]
       channel = "nightly"
-      targets = ["x86_64-unknown-none"]
+      targets = ["x86_64-unknown-none", "x86_64-unknown-uefi"]
       ```
   - Alternatively, you can use [`std::process::Command`](https://doc.rust-lang.org/stable/std/process/struct.Command.html) to invoke the build command of your kernel in the `build.rs` script.
 - Create a [`build.rs`](https://doc.rust-lang.org/cargo/reference/build-scripts.html) build script in the `os` crate. See our [disk image creation template](docs/create-disk-image.md) for a more detailed example.

--- a/examples/basic/basic-os.md
+++ b/examples/basic/basic-os.md
@@ -29,7 +29,7 @@ The top level `basic` crate defines a [workspace](https://doc.rust-lang.org/carg
     [toolchain]
     channel = "nightly"
     components = ["llvm-tools"]
-    targets = ["x86_64-unknown-none"]
+    targets = ["x86_64-unknown-none", "x86_64-unknown-uefi"]
     ```
 
 The `basic` create combines the kernel with `bootloader`, creates a bootable disk image and launches them.

--- a/examples/basic/rust-toolchain.toml
+++ b/examples/basic/rust-toolchain.toml
@@ -1,4 +1,4 @@
 [toolchain]
 channel = "nightly"
 components = ["rustfmt", "clippy", "llvm-tools"]
-targets = ["x86_64-unknown-none"]
+targets = ["x86_64-unknown-none", "x86_64-unknown-uefi"]

